README.rst

```diff
@@ -101,45 +147,61 @@ In the above image, User can see explore programs button which is pointing to ``
 Install extra requirements
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In order to install extra requirements, use DISCOVERY_EXTRA_PIP_REQUIREMENTS and re-build the docker image.::
+To install extra requirements, use DISCOVERY_EXTRA_PIP_REQUIREMENTS and re-build the docker image.
+
+.. code-block:: bash
 
-  tutor config save \
-    --set 'DISCOVERY_EXTRA_PIP_REQUIREMENTS=["git+https://github.com/myusername/myplugin"]'
+  tutor config save --set 'DISCOVERY_EXTRA_PIP_REQUIREMENTS=["git+https://github.com/myusername/myplugin"]'
 
-Then, build the image, pointing to your fork if necessary::
+Then, build the image, pointing to your fork if necessary:
+
+.. code-block:: bash
 
   tutor images build discovery
 
 Debugging
 ---------
 
-To debug the course discovery service, you are encouraged to mount the course-discovery repo from the host in the development container:
+To debug the course discovery service, you are encouraged to mount the course-discovery repo from the host
+in the development container:
+
+.. code-block:: bash
 
     tutor dev start --mount /path/to/course-discovery/ discovery
 
-You can then access the development server at http://discovery.local.edly.io:8381. Feel free to add breakpoints (``import pdb; pdb.set_trace()``) anywhere in your source code to debug your application.
+You can then access the development server at http://discovery.local.edly.io:8381. Feel free to add breakpoints
+(``import pdb; pdb.set_trace()``) anywhere in your source code to debug your application.
+
+Once a local repository is mounted in the image, you will have to install nodejs dependencies and collect static assets:
 
-Once a local repository is mounted in the image, you will have to install nodejs dependencies and collect static assets::
+.. code-block:: bash
 
     tutor dev run discovery npm install --development
     tutor dev run discovery make static.dev
 
 Troubleshooting
 ---------------
 
-This Tutor plugin is maintained by Muhammad Faraz Maqsood from `Edly <https://edly.io/>`__. Community support is available from the official `Open edX forum <https://discuss.openedx.org>`__. Do you need help with this plugin? See the `troubleshooting <https://docs.tutor.edly.io/troubleshooting.html>`__ section from the Tutor documentation.
+This Tutor plugin is maintained by Muhammad Faraz Maqsood from `Edly <https://edly.io/>`__.
```

Comment:
please use same way for adding links. You can do same as you have done on line 8 like below.
`.. _Tutor: https://docs.tutor.edly.io`.

Reply:
Added.

Reply:
Please do the same for DigitalOcean link at line 193.

Reply:
And that's added too.